data_steward/ci/upload_cloud_creds.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Run this script to upload service-account keys for gcloud on Circle CI.
+# Arg: [path to gcloud service account key JSON file]
+# Requires CI_TOKEN environment variable containing a valid Circle CI token.
+
+set -e
+
+function make_circle_envvar {
+    curl -s -X DELETE \
+        https://circleci.com/api/v1.1/project/$1/$2/$3/envvar/$4?circle-token=$CI_TOKEN
+
+    curl -s -X POST  --header "Content-Type: application/json" \
+        https://circleci.com/api/v1.1/project/$1/$2/$3/envvar?circle-token=$CI_TOKEN \
+        -d "{\"name\": \"$4\", \"value\": \"$5\"}"
+}
+
+GCLOUD_CREDENTIALS_KEY=$(openssl rand -base64 32)
+GCLOUD_CREDENTIALS=$(openssl enc  -aes-256-cbc -in $1 -base64 -A  -k $GCLOUD_CREDENTIALS_KEY)
+
+VCS_TYPE=github
+CI_USERNAME=aahladmanas
```

Comment:
We were supposed to get a system account for this. While I look into that, can you please enable `CI_USERNAME` to be specified by environment variable or script arg? Let me know if you have other ideas.

Reply:
right. cool